typespec-tests/tsp/nested-enum-discriminator.tsp

```diff
@@ -66,183 +66,183 @@ model GoblinShark extends Shark {
 @scenario
 @route("/model")
 @scenarioDoc("""
-Generate and receive polymorphic model in multiple levels inheritance with 2 discriminators.
```

Comment:
Is there any change to this file? Or just format?

Reply:
Just format.

Probably due to update of `@typespec/prettier-plugin-typespec`